pkg/backend/httpstate/backend.go

```diff
@@ -314,15 +327,16 @@ func Login(ctx context.Context, d diag.Sink, cloudURL string, opts display.Optio
 	}
 
 	// Try and use the credentials to see if they are valid.
-	valid, err := IsValidAccessToken(ctx, cloudURL, accessToken)
+	valid, username, err := IsValidAccessToken(ctx, cloudURL, accessToken)
 	if err != nil {
 		return nil, err
 	} else if !valid {
 		return nil, errors.Errorf("invalid access token")
 	}
 
 	// Save them.
-	if err = workspace.StoreAccessToken(cloudURL, accessToken, true); err != nil {
+	account := workspace.Account{AccessToken: accessToken, Username: username}
```

Comment:
Since we just verified the access token, shouldn't we be setting `LastUsed` on `workspace.Account` here?

Reply:
Yep--good catch.